sailon/random_novelty_detector.py

```diff
@@ -10,10 +8,55 @@
 
 from smqtk_core import Configurable, Pluggable
 
+from .errors import RoundError
+from .novelty_interface import ONDAdapter
+
+
 logger = logging.getLogger(__name__)
 
 
-class RandomNoveltyDetector(Configurable, Pluggable):
+class RandomNoveltyDetectorAdapter(ONDAdapter):
+    def __init__(self):
+        self.detector = RandomNoveltyDetector()
+
+    def get_config(self):
+        """
+        Implementation of smqtk Algorithm abstract method.
+        """
+        return {}
+
+    def initialize(self, config_params: dict) -> None:
```

Comment:
Could you add docstrings to this method and the other methods for the adapter?

Reply:
Added docstrings to the abstract class methods.